xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_stream.h

```diff
@@ -78,6 +78,9 @@ extern "C"
     /// @brief To identify @ref cpp_kodi_addon_pvr_Defs_Stream_PVRStreamProperties as Radio RDS.
     PVR_CODEC_TYPE_RDS,
 
+    /// @brief To identify @ref cpp_kodi_addon_pvr_Defs_Stream_PVRStreamProperties as Radio ID3.
+    PVR_CODEC_TYPE_ID3,
```

Comment:
Latest force-pushed commit does not contain the api version bump anymore. As you're appending a new enum value in between existing values here, both PVR api version and PVR api min version need to be bumped.

Reply:
Re-added, will force push after looking at the rest of the comments/problems